src/app/Plans/components/Wizard/helpers.tsx

```diff
@@ -235,6 +236,7 @@ export const filterAndConvertInventoryTree = (
       ),
       customBadgeContent: badge,
       hasBadge: !!badge,
+      defaultExpanded: true,
```

Comment:
Ha! I didn't realize it would be so simple.

Reply:
Yeah once the `defaultAllExpanded` and `allExpanded` props are out of the picture its easy ;)

Reply:
But devil hides in details, as you mentioned this doesn't solve the initial requirement.